projects/docs/docs/commands/graph.md

```diff
@@ -86,7 +86,7 @@ When the `--format` argument is specified as `json`, the following JSON schema i
 | Key       | Description                                                                                                             |
 | --------- | ----------------------------------------------------------------------------------------------------------------------- |
 | `name`    | Name of the target.                                                                                                     |
-| `product` | The product type of the target. See (Product)[https://docs.tuist.io/manifests/project#product] for all possible values. |
+| `product` | The product type of the target. See (Product)[https://docs.tuist.io/manifest-api-reference#product] for all possible values. |
```

Comment:
shouldn't this point to `https://tuist.github.io/tuist/main/documentation/projectdescription/product`? same for the other links